packages/calcite-components/src/components/tab-title/tab-title.e2e.ts

```diff
@@ -338,153 +347,44 @@ describe("calcite-tab-title", () => {
-        page = await newE2EPage();
-      });
+    expect(await tabTitleEl.getProperty("scale")).toBe(testCase.scale);
 
-      it("should inherit default medium scale from tab-nav", async () => {
-        await page.setContent(
-          html`
-            <calcite-tab-nav scale="m">
-              <calcite-tab-title selected>Tab Title</calcite-tab-title>
-              <calcite-tab-title>Tab 2 Title</calcite-tab-title>
-            </calcite-tab-nav>
-          `
-        );
-
-        const tabTitleNav = await page.find("calcite-tab-nav");
-        const tabTitleEl = await page.find("calcite-tab-title");
-        const content = await page.find(`calcite-tab-title >>> .${CSS.content}`);
-        const contentStyles = await content.getComputedStyle();
-
-        expect(tabTitleEl).toEqualAttribute("scale", "m");
-        expect(tabTitleNav).toEqualAttribute("scale", "m");
-        expect(contentStyles.fontSize).toEqual("14px");
-        expect(contentStyles.lineHeight).toEqual("16px");
-      });
+    expect(contentStyles.fontSize).toEqual(testCase.styles.fontSize);
+    expect(contentStyles.lineHeight).toEqual(testCase.styles.lineHeight);
+  }
 
-      it("should inherit small scale from tab-nav", async () => {
-        await page.setContent(
-          html`
-            <calcite-tab-nav scale="s">
-              <calcite-tab-title selected>Tab Title</calcite-tab-title>
-              <calcite-tab-title>Tab 2 Title</calcite-tab-title>
-            </calcite-tab-nav>
-          `
-        );
-
-        const tabTitleNav = await page.find("calcite-tab-nav");
-        const tabTitleEl = await page.find("calcite-tab-title");
-        const content = await page.find(`calcite-tab-title >>> .${CSS.content}`);
-        const contentStyles = await content.getComputedStyle();
-
-        expect(tabTitleEl).toEqualAttribute("scale", "s");
-        expect(tabTitleNav).toEqualAttribute("scale", "s");
-        expect(contentStyles.fontSize).toEqual("12px");
-        expect(contentStyles.lineHeight).toEqual("16px");
-      });
-
-      it("should inherit large scale from tab-nav", async () => {
-        await page.setContent(
-          html`
-            <calcite-tab-nav scale="l">
-              <calcite-tab-title selected>Tab Title</calcite-tab-title>
-              <calcite-tab-title>Tab 2 Title</calcite-tab-title>
-            </calcite-tab-nav>
-          `
-        );
-
-        const tabTitleNav = await page.find("calcite-tab-nav");
-        const tabTitleEl = await page.find("calcite-tab-title");
-        const content = await page.find(`calcite-tab-title >>> .${CSS.content}`);
-        const contentStyles = await content.getComputedStyle();
-
-        expect(tabTitleEl).toEqualAttribute("scale", "l");
-        expect(tabTitleNav).toEqualAttribute("scale", "l");
-        expect(contentStyles.fontSize).toEqual("16px");
-        expect(contentStyles.lineHeight).toEqual("20px");
-      });
-    });
-  });
+  describe("scale property", () => {
```

Comment:
Sorry for not catching this earlier, but let's drop this test entirely in favor of a screenshot test.